src/components/core/ProductTile.vue

```diff
@@ -21,7 +21,10 @@ export default {
   props: ['product'],
   computed: {
     thumbnail () {
-      return thumbnail(this.product.image, 310, 300)
+      // get first configurable child image
+      // NOTE: might not work when product has no children
+      // todo: play with the image based on category page filters - eg. when 'red' color is chosen, image is going to be 'red'
+      return thumbnail(this.product.configurable_children[0].image, 310, 300)
```

Comment:
It's only an example how ProductTile can look like - OK; but we should provide an condition here; if there are no configurable_children let's use product.image? Why not? @kacperknapik